tests/dummy/app/pods/components/demo-palette/component.js

```diff
@@ -13,7 +13,9 @@ function rgb2hex (rgb) {
 
   rgb = rgb.match(/^rgb\((\d+),\s*(\d+),\s*(\d+)\)$/)
 
-  return `#${hex(rgb[1])}${hex(rgb[2])}${hex(rgb[3])}`
+  return hex(rgb[1]) === '0'
```

Comment:
Would we run the risk of `rgb[2]` or `rgb[3]` being `0` as well? Instead should it be something like:

```js
const hex = rgb.slice(1)
  .map((value) => value === '0' ? '00' : `${hex(value)}`)
  .join('')

return `#${hex}`
```